CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,89 @@
+# Changelog 1.13
+
+## 1.13.0 - 2023-07-24
+
+### Added
+
+- Add AdminNetworkPolicy support in Antrea to align with K8s NetworkPolicy API, and document the introduction and usage. ([#5170](https://github.com/antrea-io/antrea/pull/5170) [#5270](https://github.com/antrea-io/antrea/pull/5270), [@Dyanngg])
+- Containerize Windows userspace OVS processes in OVS container and run them in a container of the Antrea Agent Pod to align with the design of Linux. ([#4936](https://github.com/antrea-io/antrea/pull/4936) [#5052](https://github.com/antrea-io/antrea/pull/5052) [#5071](https://github.com/antrea-io/antrea/pull/5071), [@rajnkamr] [@Atish-iaf] [@NamanAg30])
+- Add support of TLS, HTTP, and HTTPs protocols for Flow Aggregator to connect to the ClickHouse DB, and allow users to specify the CA certificate for TLS and HTTPs. ([#5171](https://github.com/antrea-io/antrea/pull/5171), [@yuntanghsu])
+- Add Egress information to flow records for Pod-to-external flows in Flow Exporter. ([#5088](https://github.com/antrea-io/antrea/pull/5088), [@dreamtalen])
+- Increase accuracy of Pod information in the flow records by adding a Pod store in Flow Exporter and Flow Aggregator for them to fetch the Pod information. ([#5185](https://github.com/antrea-io/antrea/pull/5185), [@yuntanghsu])
+- Add support to consume Service annotation `service.kubernetes.io/topology-mode` on AntreaProxy since the old `service.kubernetes.io/topology-aware-hints` annotation has been deprecated in Kubernetes 1.27. ([#5241](https://github.com/antrea-io/antrea/pull/5241), [@mengdie-song])
+- Add a new Prometheus metric to represent the number of packets dropped by OVS meter. ([#5165](https://github.com/antrea-io/antrea/pull/5165), [@mengdie-song])
+- Add support for the `sort-by` flag in more `antctl get` commands for more fields. ([#4346](https://github.com/antrea-io/antrea/pull/4346), [@jainpulkit22])
+- Add `kubeAPIServerOverride` option to allow user to override the address of Kubernetes apiserver for antrea-controller. ([#5056](https://github.com/antrea-io/antrea/pull/5056), [@tnqn])
+- Add a configuration toggle for FlowExporter to prevent Antrea Agent from showing error log `Error when initializing flow exporter` when user enable FlowExporter feature gate without installing Flow Aggregator. ([#5021](https://github.com/antrea-io/antrea/pull/5021), [@yuntanghsu])
```

Comment:
Probably just say: "Add a FlowExporter configuration toggle to antrea-agent for users to explicitly enable/disable flow exports."

And I feel this one should be "Changed".